wiki/Contributor's-Guide.md

```diff
@@ -96,7 +96,7 @@ Make sure you fully read through the above [Contributor's Guide](#contributors-g
 
 To get an idea of what Drasil can do, have a look through the [Drasil website](https://jacquescarette.github.io/Drasil/). The website contains links to Software Requirement Specification (SRS) documents, both in an HTML and PDF format. There are also some links to generated code along with a case studies table to demonstrate the variety of choices that can be made by the user at the time of code generation. There are links to documentation within Drasil, as well as a section analyzing Drasil (we'll get to these last two sections later). All of the examples, example code, and the website itself are artifacts generated by Drasil. You can see their source code in the `drasil-example` and `drasil-website` folders respectively. 
 
-One of the most important things in working with Drasil is having a solid foundation for using GitHub. Read through [Git2Know](Git2Know) to learn more about branching, pull requests, merging, and anything else git or Github related. Also take a look at the [Makefile documentation](Makefile), since that will be the primary way you compile and use Drasil. More optionally, you may want to take a brief look through [Creating Your Project in Drasil](Creating-Your-Project-in-Drasil), and some [tips for debugging](Debugging-in-Drasil).
+One of the most important things in working with Drasil is having a solid foundation for using GitHub. Read through [Git2Know](https://github.com/JacquesCarette/Drasil/wiki/Git2Know-for-Drasil) to learn more about branching, pull requests, merging, and anything else git or Github related. Also take a look at the [Makefile documentation](Makefile), since that will be the primary way you compile and use Drasil. More optionally, you may want to take a brief look through [Creating Your Project in Drasil](Creating-Your-Project-in-Drasil), and some [tips for debugging](Debugging-in-Drasil).
```

Comment:
I think you can omit the `https://..../` part -- we should only need `Git2Know-for-Drasil`, no?